engine/redis/engine.go

```diff
@@ -28,7 +28,7 @@ type Engine struct {
 	timer   *Timer
 	meta    *MetaManager
 	monitor *SizeMonitor
-	storage storage.Storage
+	storage persistence.Persistence
```

Comment:
Is it a little bit confusing here between variable name and type?

Reply:
Yes, you're right. Should move the `Persistence` interface out of package persistence. Just wait a minute.